package.json

```diff
@@ -110,6 +110,9 @@
     "webpack": "^5.93.0",
     "webpack-dev-server": "^5.0.4"
   },
+  "overrides": {
+    "postcss": "^8.4.39"
```

Comment:
I can't find why we should do like this for which module update.
Could you be more specific?

Reply:
OK, I updated in PR description